### DISCUSSION TITLE
Welcome to Zettel Notes Discussions!

### DISCUSSION THREAD

**Discussion by damionx7** [damionx7/Zettel-Notes-Documentation#51] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share and upvote suggestions and feature requests.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by chrislou1961** [comment]
Is there a way to add tags to multiple notes at once?

**Reply by damionx7** [reply]
Hey, Its not possible. Pls create a feature request if you want this feature. We have a **maintenance section** in app, where all housekeeping tasks will be performed. Some ideas include:
1. Removing whitespaces and extra lines
2. Removing repeated tags from notes
3. Removing redundant links in files
